sql/hive/pom.xml

```diff
@@ -36,6 +36,12 @@
   </properties>
 
   <dependencies>
+    <!-- Added for sbt issue, expected to be evicted -->
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>2.10.3</version>
```

Comment:
This doesn't match the 2.10.4 used in the build; this should be `${scala.version}`?
CC @vanzin since I think he responded about a similar problem but resolved it without a build change -- just nuked the ivy2 directory or something.